app/handlers/documents.py

```
@@ -113,15 +113,22 @@ async def _process_telegram_file(
 
     try:
         logger.info("[documents] Отправляю файл в МС: path=%s kind=%s", dest, log_kind)
```

Ruff (0.14.2)

String contains ambiguous `М` (CYRILLIC CAPITAL LETTER EM). Did you mean `M` (LATIN CAPITAL LETTER M)?

(RUF001)